compiler/daml-lf-reader/BUILD.bazel

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+load("//bazel_tools:haskell.bzl", "da_haskell_library")
+
+da_haskell_library(
```

Comment:
Why not just shove this in the daml-lf-ast library? And perhaps rename that daml-lf

Reply:
I think the proto generated vs hand written split makes sense, but from then on, i just want a daml-lf library. I don't feel too strongly though.

Reply:
We already have `daml-lf-proto`, `daml-lf-ast`, and `daml-lf-tools`. I don’t really care where it ends up but given that we have a PR blocked on this fix, I would like to merge it without merging it for now.

Reply:
Certainly